src/test/scala/kamon/statsd/SimpleMetricKeyGeneratorSpec.scala

```diff
@@ -42,6 +42,14 @@ class SimpleMetricKeyGeneratorSpec extends WordSpec with Matchers {
       buildMetricKey("trace", Map("metric-name-1" -> "POST: /kamon/example", "metric-name-2" -> "elapsed-time")) should be("kamon.localhost.trace.POST-_kamon_example.elapsed-time")
     }
 
+    "generate metric names without tags that follow the application.host.entity.entity-name.metric-name pattern by default" in {
```

Comment:
These tests might change depending on how we resolve the metric + tags to StatsD key thing.